src/rest/connector/__init__.py

```diff
@@ -16,6 +16,9 @@
 from genie.abstract import Lookup
 import rest.connector.libs
 
+# Ease of use (rest.connector.Acisdk)
+from rest.connector.acisdk_implementation import AciCobra
```

Comment:
rename to AciCobra to ACIsdk

Reply:
After discussing with @dbasavar, we noted an alternate way for the file / class naming:
there would be possible to remove the __init__ class initialization, rename the file to "aci.py" and class to "Cobrasdk" (thus the connector could be initialized in the template using rest.connector.aci.Cobrasdk, versus the current rest.connector.Acisdk).
Kindly suggest which version you would prefer.